app/styles/cards.module.css

```diff
@@ -214,7 +214,9 @@
   }
 
   .card {
-    padding: 20px 10px;
+    padding: 20px 20px;
+    width: 90%;
+    align-items: normal;
```

Comment:
Why .card has been changed?
If somewhere else this class has been used it will break the UI there as it is general class which I told you I guess